### ISSUE TITLE
bug: may be problem, some highlight changed in "dawn" variant

### ISSUE THREAD

**Issue by nanozuki** [rose-pine/neovim#206]
### Neovim version (nvim -v)

NVIM v0.9.5 Build type: Release LuaJIT 2.1.1693350652

### Terminal / multiplexer

kitty

### Describe the bug

1. The seprator's background between windows seems wrong, not same with windows' background:
    <img width="126" alt="图片" src="https://github.com/rose-pine/neovim/assets/4208028/511a9a2f-3a6c-47a7-9896-30b597d92b70">
1. Color changed in [NeoTree](https://github.com/nvim-neo-tree/neo-tree.nvim). Became more black and gray. Is this expective change or bug?
    <img width="380" alt="图片" src="https://github.com/rose-pine/neovim/assets/4208028/84b361a4-135c-4b29-bdab-a8933ad987f9">
    <img width="331" alt="图片" src="https://github.com/rose-pine/neovim/assets/4208028/e1ba288c-7088-46ac-8f23-858a83826792">

### Repro

```Lua
vim.o.packpath = "/tmp/nvim/site"

local plugins = {
	rose_pine = "https://github.com/rose-pine/neovim",
	-- ADD OTHER PLUGINS _NECESSARY_ TO REPRODUCE THE ISSUE
}

for name, url in pairs(plugins) do
	local install_path = "/tmp/nvim/site/pack/test/start/" .. name
	if vim.fn.isdirectory(install_path) == 0 then
		vim.fn.system({ "git", "clone", "--depth=1", url, install_path })
	end
end

require("rose-pine").setup({
	require("rose-pine").setup({ variant = "dawn" }),
})

vim.cmd("colorscheme rose-pine")
```

**Comment by mvllow** [comment]
1. Fixed window separator in #208.
2. Reverted palette changes in #207. We are experimenting with higher contrast for a few colours but will continue testing on the canary branch instead. Would love feedback on if you like or don't like the darker text in dawn, though.

Thanks for reporting :)

**Comment by nanozuki** [comment]
Thank you for your work.

Regarding the color palette change, I'm very used to the warm and colorful feel of the previous dawn variant. Since this update has many changes, I doubt it's a bug.

But I also think making different attempts, such as increasing contrast, is necessary and beneficial for this project, so please continue. Maybe add a option to alter the contrast is also a way.
